torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -1024,6 +1024,46 @@ void CudaCodeGen::Initialize() {
     thread_block_size_ = -1;
   }
 
+  // Build an LLVM based eval expression for the extents
+  block_extents_eval_.reserve(block_extents.size());
+  std::vector<BufferArg> extents_buffer_args;
+  std::unordered_set<VarPtr> vars_in_extents;
+  for (const auto& be : block_extents) {
+    auto v = VarFinder::find(be);
+    vars_in_extents.insert(v.begin(), v.end());
+  }
+  for (const auto& te : thread_extents) {
+    auto v = VarFinder::find(te);
+    vars_in_extents.insert(v.begin(), v.end());
+  }
+  for (const size_t i : c10::irange(buffer_args.size())) {
+    if (vars_in_extents.count(buffer_args[i].var())) {
+      extents_buffer_args.push_back(buffer_args[i]);
+      arg_pos_in_extents_.push_back(true);
```

Comment:
Add `.reserve` for this.

Reply:
this is in initialize, not hot path, so perf doesn't matter as much here. but i can add